code/controllers/subsystem/processing/instruments.dm

```diff
@@ -4,16 +4,26 @@ PROCESSING_SUBSYSTEM_DEF(instruments)
 	init_order = INIT_ORDER_INSTRUMENTS
 	flags = SS_KEEP_TIMING
 	priority = FIRE_PRIORITY_INSTRUMENTS
-	var/static/list/datum/instrument/instrument_data = list()		//id = datum
+	/// List of all instrument data, associative id = datum
+	var/static/list/datum/instrument/instrument_data = list()
```

Comment:
```suggestion
	var/list/datum/instrument/instrument_data = list()
```
There is a single instruments subsystem, so why make this and the other variables static?

Reply:
@Rohesie in theory this means that I don't have to do Recover() as it'll just be there

Reply:
I suppose that makes sense.